### DISCUSSION TITLE
Does the code support different material parameters for each tetrahedron?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#1006] [Q&A] [answered]
I found that isotropy defines material parameters in the following way, how does the code implement anisotropic simulation, can you set different material parameters for each tetrahedron separately?
![12a19151dfaed7890d519a279b044e6](https://github.com/SeisSol/SeisSol/assets/114474894/d35ec9fe-9183-4ef3-bc5e-46181c5a7f6e)

**Comment by NicoSchlw** [comment, marked as answer]
Hi, this docs page describes how you set up anisotropic materials: https://seissol.readthedocs.io/en/latest/physical-models.html#anisotropic. To include 3D heterogeneous material parameters, use an ASAGI file: https://seissol.readthedocs.io/en/latest/asagi.html.
